README.md

```diff
@@ -8,7 +8,32 @@ Okta Spring Boot Starter
 
 Okta's Spring Boot Starter will enable your Spring Boot application to work with Okta via OAuth 2.0/OIDC.
 
-**NOTE:** This library works with Spring Boot 2.2+. If you need support for Spring Boot 1.5.x, use version 0.6.
+**NOTE:** This library works with Spring Boot 2.4+. If you need support for Spring Boot 1.5.x, use version 0.6.
+
+## Release status
+
+This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/).
+
+:heavy_check_mark: The current stable major version series is: 1.5.x
+
+| Version | Status                    |
+| ------- | ------------------------- |
+| 0.x.x | :warning: Retired |
+| 1.0.0 - 1.3.0 | :warning: Retired |
+| 1.4.0 | :clock9: Retiring effective August 20, 2021 |
+| 1.5.0 | :heavy_check_mark: Stable |
+
+## Spring Boot Compatibility
+
+| Okta Spring Boot SDK Version | Compatible Spring Boot Version |
+| ------- | ------------------------- |
+| 0.6 | 1.5.x |
+| 1.0.0, 1.1.0 | 2.1 |
+| 1.2.x | 2.2.0.M3 |
+| 1.3.0, 1.4.0 | 2.2.0.RELEASE |
+| 1.5.0 | 2.4.0 |
```

Comment:
We could just copy the ranges from: https://github.com/spring-io/start.spring.io/blob/17470056557679f0641f6bf653bcddacc53eb962/start-site/src/main/resources/application.yml#L427-L439